docs/command/atlas-api-projectIpAccessList-createProjectIpAccessList.txt

```diff
@@ -104,3 +104,29 @@ Inherited Options
      - false
      - Name of the profile to use from your configuration file. To learn about profiles for the Atlas CLI, see https://dochub.mongodb.org/core/atlas-cli-save-connection-settings.
 
+Examples
+-----------------
+
+.. tabs::
+   .. tab:: Add Entries to Project IP Access List
+      :tabid: 2023-01-01_project_ip_access_list_add
+      Adds multiple access list entries to the specified project
+
+      .. code-block::
```

Comment:
This differs from the example where we instruct the user to create a file and "run the commands commands". Could we make it more similar?

I'm not sure that your example is cross-platform. (Thinking about windows)
```bash
cat <<EOF > payload.json
            [
              {
                "cidrBlock": "192.168.1.0/24",
                "comment": "Internal network range"
              },
              {
                "cidrBlock": "10.0.0.0/16",
                "comment": "VPC network range"
              }
            ]
         EOF
```